waterloop-site/src/components/SustainableTech/Teams/hooks/team-images.ts

```diff
@@ -0,0 +1,95 @@
+import React, { useCallback, useState } from 'react';
+
+import goose1 from '../../../../static/img/pgimgs/goose1.png';
+import goose2 from '../../../../static/img/pgimgs/goose2.png';
+import goose3 from '../../../../static/img/pgimgs/goose3.png';
+import goose4 from '../../../../static/img/pgimgs/goose4.png';
+import TeamPic from '../../../../static/img/team/team.png';
+
+interface Image {
+  imgFile: string;
+  name: string;
+  desc: string;
+}
+
+interface TeamImage {
+  image: string;
+  name: string;
+  desc: string;
+  imgs: Image[];
+  currentTeam: number;
+  cycleRight: React.EffectCallback;
+  cycleLeft: React.EffectCallback;
+  selectTeam: (index: number) => () => void;
+}
+
+type TeamImagesHook = () => TeamImage;
+
+const imgs: Image[] = [
```

Comment:
Instead of storing this data here, for slightly better reusability and better boundaries, I would store this data in `waterloop-site/src/static/copy/SustainableTech/Teams.json`, import it over, and then cast it to the `Image[]` type. I would also place the `imgFile` file paths directly each object in the array.
Refer to my PR for clarification on what I mean: https://github.com/waterloop/waterloop-website-w20/pull/125/files